tools/crm_resource_runtime.c

```diff
@@ -1308,7 +1309,9 @@ cli_resource_restart(pcmk__output_t *out, pe_resource_t *rsc, const char *host,
     }
 
     rsc_id = strdup(rsc->id);
-    if ((pe_rsc_is_clone(rsc) || pe_bundle_replicas(rsc)) && host) {
+    lookup_id = clone_strip(rsc->id);
```

Comment:
We don't want to do this if rsc is a unique clone instance (uber_parent() is a clone and pcmk_is_set(rsc->flags, pe_rsc_unique))

Reply:
Conveniently, we have pe_rsc_is_unique_clone for testing this.  Note that I haven't tested unique clones at all for this PR.  I guess I need to add that to my cluster and see what happens.